popups.js

```diff
@@ -0,0 +1,414 @@
+      height: '100%',
+      width: '100%',
+      position: 'fixed',
+      top: 0,
+      left: 0,
+      zIndex: 9,
+      overflow: 'auto',
+      padding: '16px',
+    };
+    Object.assign(modal.style, modalPty);
+
+    const card = document.createElement('div');
+    const cardPty = {
+      backgroundColor: 'white',
+      width: '100%',
+      height: '100%',
+      borderRadius: '10px',
+      padding: '12px',
+      opacity: 1,
+    };
+    Object.assign(card.style, cardPty);
+
+    modal.appendChild(card);
+
+    const title = document.createElement('h2');
+    title.textContent = projects[i].Title;
+    const titlePty = {
+      fontFamily: "'Poppins', sans-serif",
+      fontStyle: 'normal',
+      fontWeight: 700,
+      fontSize: '40px',
+      color: '#172b4d',
+    };
+    Object.assign(title.style, titlePty);
+
+    const closeButt = document.createElement('span');
+    closeButt.innerHTML = projects[i].closeButt;
+    const closeButtPty = {
+      color: '#67798e',
+      fontSize: '24px',
+      fontWeight: '900',
+      position: 'absolute',
+      top: '35px',
+      right: '32px',
+    };
+    Object.assign(closeButt.style, closeButtPty);
+
+    card.append(title, closeButt);
+
+    const counter1Img = document.createElement('img');
+    counter1Img.src = projects[i].counter1;
+    counter1Img.style.marginLeft = '5px';
+    counter1Img.style.marginRight = '5px';
+
+    const counter2Img = document.createElement('img');
+    counter2Img.src = projects[i].counter2;
+    counter2Img.style.marginLeft = '5px';
+    counter2Img.style.marginRight = '5px';
+
+    const positions = document.createElement('div');
+    const positionsPty = {
+      fontFamily: "'Poppins', sansSerif",
+      fontStyle: 'normal',
+      fontWeight: 600,
+      fontSize: '13px',
+      color: '#344563',
+      margin: '12px 0',
+    };
+    Object.assign(positions.style, positionsPty);
+
+    positions.append(projects[i].Client, counter1Img, projects[i].Role, counter2Img,
+      projects[i].year);
+
+    card.appendChild(positions);
+
+    const snapshot = document.createElement('img');
+    snapshot.src = projects[i].snapshot;
+
+    card.append(snapshot);
+
+    const description = document.createElement('p');
+    description.textContent = projects[i].description;
+    description.style.marginTop = '12px';
+    description.style.marginBottom = '24px';
+    card.append(description);
+
+    const badges = document.createElement('ul');
+
+    for (let j = 0; j < projects[i].badges.length; j += 1) {
+      const li = document.createElement('li');
+      li.textContent = projects[i].badges[j];
+
+      const liPty = {
+        padding: '4px 12px',
+        background: '#ebebff',
+        borderRadius: '8px',
+        fontFamily: "'Poppins', sans-serif",
+        fontStyle: 'normal',
+        fontWeight: 500,
+        fontSize: '12px',
+        letterSpacing: '0.03em',
+        color: '#6070ff',
+        margin: '5px',
+      };
```

Comment:
- [ ] I would kindly suggest you stick to the normal way of styling using the external CSS file for now for easy accessibility and navigation 👍 👍